### PR TITLE
Renames js article about advantages

### PR DESCRIPTION
**Pull request by Rafase282**

### CODE REVIEW

File: Online-VS-Offline-IDE.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ There are several offline [IDE](Integrated-Development-Environment) that program
...
-- **[Cloud9 IDE](https://c9.io)** Cloud9 IDE is a open source, from version 3.0, online integrated development environment. It supports hundreds of programming languages, including [PHP](PHP), [Ruby](Ruby), Perl, [Python](Python), [JavaScript](Advantages-Disadvantages-Javascript) with Node.js, and [Go](Go). It enables developers to get started with coding immediately with pre-setup workspaces, collaborate with their peers with collaborative coding features, and web development features like live preview and browser compatibility testing.
+- **[Cloud9 IDE](https://c9.io)** Cloud9 IDE is a open source, from version 3.0, online integrated development environment. It supports hundreds of programming languages, including [PHP](PHP), [Ruby](Ruby), Perl, [Python](Python), [JavaScript](JS-Advantages-Disadvantages) with Node.js, and [Go](Go). It enables developers to get started with coding immediately with pre-setup workspaces, collaborate with their peers with collaborative coding features, and web development features like live preview and browser compatibility testing.
```
Comment (abhisekp) [inline Online-VS-Offline-IDE.md]:
```diff
- **[Cloud9 IDE](https://c9.io)**
+ [**Cloud9 IDE**](https://c9.io)
```

**Comment by Rafase282** [comment]
@abhisekp **be bold!**

Made changes and some more.

File: Online-VS-Offline-IDE.md [hunk 2]
```diff
@@ -2,29 +2,31 @@
 
 ## Offline IDEs
 
-There are several offline [IDE](Integrated-Development-Environment) that programmers use in order to stay more productive to learn and be more productive at work.
+There are several offline [**IDE**](Integrated-Development-Environment) that programmers use in order to stay more productive to learn and be more productive at work.
 
-- **[Eclipse](https://www.app5.unisys.com/offerings/ClearPathConnection/eclipse.html)** is more than a Java editor. Its biggest advantages are code completion by tabbing method which saves a lot of time when writing documentation.
+- [**Eclipse***](https://www.app5.unisys.com/offerings/ClearPathConnection/eclipse.html) is more than a Java editor. Its biggest advantages are code completion by tabbing method which saves a lot of time when writing documentation.
```
Comment (abhisekp) [inline Online-VS-Offline-IDE.md]:
```diff
- [**Eclipse***]
+ [**Eclipse**]
```

File: Online-VS-Offline-IDE.md [hunk 3]
```diff
@@ -2,29 +2,31 @@
...
 
   It has a built-in syntax check to fix any mistype words while coding a project. All of features we've come to expect of an IDE are their code completion, templates, integration  with various SCMSs, and integration with build systems.
 
   Its code formatting and cleanup tools are very powerful. I find its build system to work well and intuitively. I think these are the things upon which its reputation is really built.
 
   Furthermore it has a refactoring that find and replace function, variables and classes of a specific work.
 
-- **[NetBeans](https://netbeans.org/features/)** is a free intuitive editor that does it all compared from other IDEs. It has simple Swing GUI design tool to build user interfaces by dragging and dropping components for instance buttons, textboxes.
+- [**NetBeans***](https://netbeans.org/features/) is a free intuitive editor that does it all compared from other IDEs. It has simple Swing GUI design tool to build user interfaces by dragging and dropping components for instance buttons, textboxes.
```
Comment (abhisekp) [inline Online-VS-Offline-IDE.md]: same here as [above :point_up:](https://github.com/FreeCodeCamp/wiki/pull/991/files#diff-633c89bc39d158b1787c8b7db297a536R7)

**Pull request merged**